### PR TITLE
Include py.typed marker in sdist/wheel packages

### LINKED ISSUE #161: py.typed marker missing from release breaks type checking
**Describe the bug**

while this package contains type annotations, the released files don't contain a `py.typed` file (see [PEP 561](https://www.python.org/dev/peps/pep-0561/)), which means type checkers such as mypy are not able to properly check code using this library.

**To Reproduce**

- Install the package 🙃
- `.../site-packages/UnleashClient/py.typed` does not exist

**Expected behavior**

`.../site-packages/UnleashClient/py.typed` exists

### PR DESCRIPTION
**Pull request by wbolster**
This marks the package as ‘typing friendly’ and enables tools like mypy
to actually type check code using this library.

Closes #161.

### CODE REVIEW

**Comment by wbolster** [comment]
proof that it works:

``` shell
$ make build-package 

$ unzip -l dist/UnleashClient-4.4.0-py3-none-any.whl | grep -F py.typed
        0  2021-06-25 11:13   UnleashClient/py.typed

$ tar tf dist/UnleashClient-4.4.0.tar.gz | grep -F py.typed
UnleashClient-4.4.0/UnleashClient/py.typed
```

**Comment by wbolster** [comment]
broken ci is not caused by these changes. see #163

**Comment by ivarconr** [comment]
Thanks. This is a nice improvement. I will wait for feedback from @ivanklee86  before we merge it ;)

**Review by ivarconr — APPROVED** [review]

**Comment by ivanklee86** [comment]
Woo!  Thanks @wbolster! :)

**Comment by wbolster** [comment]
yw :-1: this would need a pypi release to be actually useful though hint hint wink wink :smile:

**Pull request merged**
